source/codegen/metadata/nirfmxspecan/attributes.py

```diff
@@ -75,6 +75,11 @@
         'name': 'LIMITED_CONFIGURATION_CHANGE',
```

Comment:
This change itself looks fine but doesn't seem to match  the scrapigen change. What happened with that?